lib/storage/src/content_manager/conversions.rs

```diff
@@ -62,6 +62,7 @@ impl TryFrom<api::grpc::qdrant::CreateCollection> for CollectionMetaOperations {
                     .sharding_method
                     .map(sharding_method_from_proto)
                     .transpose()?,
+                strict_mode_config: value.strict_mode_config.map(|i| i.into()),
```

Comment:
`From::from` has better navigation from rust-analyzer :)
```suggestion
                strict_mode_config: value.strict_mode_config.map(From::from),
```